packages/grafana-ui/src/components/Select/SelectOptionGroup.tsx

```diff
@@ -81,11 +79,11 @@ class UnthemedSelectOptionGroup extends PureComponent<ExtendedGroupProps, State>
 
     return (
       <div>
-        {/* TODO: fix keyboard a11y */}
-        {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
-        <div className={styles.header} onClick={this.onToggleChildren}>
+        {/*React Select doesn't support focusable option group headers, this will be skipped when using
+      the keyboard */}
+        <div className={styles.header} onClick={this.onToggleChildren} role="presentation">
```

Comment:
pretty sure this is the only change that's needed right? every other change can be undone

Reply:
yes, you are right, I will undo the other changes.